libs/utility_manager.py

```
@@ -43,7 +43,7 @@ def _open_resource_file(self, filename):
 		try:
 			if os.path.isfile(filename):
 				if platform.system() == "Windows":
-					subprocess.call(['start', filename], shell=True)
+					os.startfile(filename)
```

Ruff (0.15.15)

[error] Starting a process without a shell

(S606)